test/EntityFramework.Sqlite.Tests/Migrations/SqliteMigrationSqlGeneratorTest.cs

```
@@ -47,33 +63,6 @@ public void RestartSequenceOperation_not_supported()
             Assert.Equal(Strings.SequencesNotSupported, ex.Message);
         }
 
-        public override void AddColumnOperation_with_defaultValue()
-        {
-            base.AddColumnOperation_with_defaultValue();
-
-            Assert.Equal(
-                @"ALTER TABLE ""People"" ADD ""Name"" varchar(30) NOT NULL DEFAULT 'John Doe';" + EOL,
-                Sql);
-        }
-
-        public override void AddColumnOperation_with_defaultValueSql()
-        {
-            // Override base test because CURRENT_TIMESTAMP is not valid for AddColumn
-            Generate(
-                new AddColumnOperation
-                {
-                    Table = "People",
-                    Name = "Age",
-                    Type = "int",
-                    IsNullable = true,
-                    DefaultExpression = "10"
-                });
-
-            Assert.Equal(
-                @"ALTER TABLE ""People"" ADD ""Age"" int DEFAULT (10);" + EOL,
-                Sql);
-        }
-
```

We decided to pass the SQL through to the database and let it throw. These test that we're generating the "correct" (or at least valid-to-parse) SQL.